### PR TITLE
Create consent_screen_ui.gs

### PR DESCRIPTION
**Pull request by siddhimahawar**

### CODE REVIEW

File: consent_screen_ui.gs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  var headerKeyValue = CardService.newKeyValue()
+                           .setContent('<b>Consent Screen</b>')
+                           .setButton(proceedImageButton);
+
+  tabSection.addWidget(headerKeyValue);
+  card.addSection(tabSection);
+  var section2 = CardService.newCardSection();
+
+  // Add the time Filter
```
Comment (shikhagupta1212) [inline consent_screen_ui.gs]: fix comment.

File: consent_screen_ui.gs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  card.addSection(contentSection);
+
+  var proceedButton = 
+      CardService.newTextButton()
+          .setText('Proceed')
+          .setOnClickAction(proceedAction)
+          .setTextButtonStyle(CardService.TextButtonStyle.FILLED);
+
+  var cancelAction = CardService.newAction()
+                         .setFunctionName('createUI');
```
Comment (shikhagupta1212) [inline consent_screen_ui.gs]: Can you add a comment, what is happening when user clicks on cancel. Also, can you show the message to user that report will not be generated.

File: consent_screen_ui.gs (hunk 1 above)
Comment (siddhimahawar) [reply consent_screen_ui.gs]: Updated

File: consent_screen_ui.gs (hunk 2 above)
Comment (siddhimahawar) [reply consent_screen_ui.gs]: Okay

Comment (siddhimahawar) [reply consent_screen_ui.gs]: Updated

**Review by piy314 — APPROVED** [review]

**Pull request merged**